solutions/search/serverless-elasticsearch-get-started.md

```diff
@@ -55,14 +55,24 @@ Use your {{ecloud}} account to create a fully-managed {{es}} project:
 6. Once the project is ready, select **Continue**.
 
 ::::{tip}
+
+**The General Purpose (GP) profile is recommended for most search use cases such as:**
+
+- full text search
+- sparse vector-based semantic search (such as when using ELSER)
+- dense vectors compressed using BBQ (enabled by default)
```

Comment:
It seems from https://www.elastic.co/docs/reference/elasticsearch/index-settings/bbq#bbq-hnsw like it's only the default in 9.1 and later. If that's correct, we either need to qualify this statement or remove the assertion about default behaviour:
```suggestion
- dense vectors compressed using BBQ
```